### DISCUSSION TITLE
Drive usage color bar options

### DISCUSSION THREAD

**Discussion by vertigo220** [doublecmd/doublecmd#1974] [Suggestions]
Instead of a solid color that's green or red without even an option of when to change, or a gradient color that doesn't make much sense as a drive can be almost full and the bar is still mostly green, it would be  preferable to have the bar be a solid color but on a gradient based on usage, with thresholds set by the user, so I could for example set it to be dark green up to 50%, then light green up to 70%, then yellow to 80%, orange to 90%, and red past that.

I'd also like to add that even though I saw this option when I was going through all the options, when I went back only a short while later to find it and change it, it took forever to find it again. It wasn't very obvious to click the "Category" drop-down, and searching for "drive color" and similar words wasn't helpful. Perhaps instead of having them under the dropdown, they should just each have their own page as a subtree under "Colors," like "File panels" and "File types."

**Comment by Skif-off** [comment]
> it took forever to find it again.

[A filter by parameter name is available at the bottom of the window: the program will display only those sections that contain a matching parameter.](https://doublecmd.github.io/doc/en/configuration.html#settings)

Or: go to the "Help" in the main menu > "2.2. Configuration" > Ctrl+F > "indicator".

**Reply by vertigo220** [reply]
Yes, but as I said, I tried that and it didn't come up. The problem with relying on search to find something is it only works if you search for the right terms. I tried combinations of "drive" and "color" and "usage," all of which seem, to me, to be obvious keywords that should find it. I didn't consider "indicator." Just seems strange to have that one page set up that way, unlike the rest of the options, and all it does is obscures the settings.

**Reply by antodc** [reply]
If I may add, I also had some difficuties in the past to find things among the Options. In the case we are taliking about, for instance, the relevant drop-down menu item is called "Drive Free Space Indicator". However, if you type either "Drive", "Free" or "Space" in the filter you do not get the menu item "Colors" in the selected items. For some reason, it only works with "Indicator". I think everyone should agree that this is a suboptimal behaviour of that filter function.
